internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,65 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// populateVolOptions update the fields in rbdVolume struct based on the request it received.
+func populateRbdVol(
+	ctx context.Context,
+	req *csi.NodeStageVolumeRequest,
+	disableInUseChecks bool,
+	volID string, cr *util.Credentials) (*rbdVolume, error) {
```

Comment:
volID is part of req, reusing avoid the number of arguments.

Reply:
done..